docs/topics/api/signing.rst

```diff
@@ -36,7 +36,7 @@ validation or review fails.
 If the upload succeeded then it will be submitted for
 validation and you will be able to check its status.
 
-.. http:put:: /api/v3/addons/[string:add-on-id]/versions/[string:version]/
+.. http:put:: /api/v3/addons/[string:addon-id]/versions/[string:version]/
```

Comment:
Since the other dash is used for a space I thought it seemed weird and we weren't being consistent. I normally opt for `add-on` but `add-on-id` seemed strange and we normally camel case as `AddonThing` so I went with it.